content/zh/blog/_posts/2019-11-26-cloud-native-java-controller-sdk.md

```diff
@@ -0,0 +1,139 @@
+---
+layout：blog
+title："使用 Java 开发一个 Kubernetes controller"
+date：2019-11-26
+slug：Develop-A-Kubernetes-Controller-in-Java
```

Comment:
please make sure yaml format in here

Reply:
ok,thanks!I will code following yaml format in futrue.